configs/fourdag/shelf_config/eval_keypoints3d_debug.py

```diff
@@ -0,0 +1,87 @@
+type = 'BottomUpAssociationEvaluation'
```

Comment:
This file shall be kept in an ignored dir, say `./xrmocap/...`. We'd better not push debug files.